guardian/manage-frontend
# Update es-lint dev dependencies to latest versions

## What does this change?
This is a housekeeping PR which:
+ Bumps some eslint-related dev dependencies to their latest versions
+ Adds `eslint-disable-next-line` comments to a number of files which trigger lint-related errors when commiting file updates locally.

## How to test
Make sure the site runs as before. No actual code has been touched in this update.
